src/static/timetable.ts

```diff
@@ -1,3 +1,3 @@
-export const TIME_STRING = ['09:00', '', '10:00', '', '11:00', '', '12:00', '', '13:00', '', '14:00', '', '15:00', '', '16:00', '', '17:00', ''];
-export const DAYS_STRING = ['월요일', '화요일', '수요일', '목요일', '금요일'];
+export const TIME_STRING = ['9', '', '10', '', '11', '', '12', '', '13', '', '14', '', '15', '', '16', '', '17', '', '18', ''];
```

Comment:
코드만 보곤 의도를 파악하기 힘든데 아래처럼 쓰면 좀 더 바꾸는 사람은 편하게 쓸 수 있을거같네여
```suggestion
// [9, '', 10, ''] 형태로 바꿉니다. ~~써야하는 이유~~~
export const TIME_STRING = ['9', '10', '11', '12', ...].flatMap((time) => [time, '']);
```

Reply:
수정했습니다~ 089859169042a88faffff596642d82949bf17a58